Documentation/WritingReST/Reference/Graphics/Diagrams.rst

```diff
@@ -12,713 +12,47 @@ PlantUML diagrams
-
-
-..  index:: Diagrams; Class
-
-Class diagram
-=============
-
-..  uml::
-
-    class Foo1 {
-        You can use
-        several lines
-        ..
-        as you want
-        and group
-        ==
-        things together.
-        __
-        You can have as many groups
-        as you want
-        --
-        End of class
-    }
-
-    class User {
-        ..  Simple Getter ..
-        + getName()
-        + getAddress()
-        ..  Some setter ..
-        + setName()
-        __ private data __
-        int age
-        -- encrypted --
-        String password
-    }
-
-..  code-block:: rst
-
-    ..  uml::
-
-        class Foo1 {
-            You can use
-            several lines
-            ..
-            as you want
-            and group
-            ==
-            things together.
-            __
-            You can have as many groups
-            as you want
-            --
-            End of class
-        }
-
-        class User {
-            ..  Simple Getter ..
-            + getName()
-            + getAddress()
-            ..  Some setter ..
-            + setName()
-            __ private data __
-            int age
-            -- encrypted --
-            String password
-        }
-
-Docs: https://plantuml.com/class-diagram
-
-..  index:: Diagrams; Component
-
-Component diagram
-=================
-
-..  uml::
-
-    package "Some Group" {
-        HTTP - [First Component]
-        [Another Component]
-    }
-
-    node "Other Groups" {
-        FTP - [Second Component]
-        [First Component] --> FTP
-    }
-
-    cloud {
-        [Example 1]
-    }
-
-    database "MySql" {
-        folder "This is my folder" {
-            [Folder 3]
-        }
-        frame "Foo" {
-            [Frame 4]
-        }
-    }
-
-    [Another Component] --> [Example 1]
-    [Example 1] --> [Folder 3]
-    [Folder 3] --> [Frame 4]
-
-..  code-block:: rst
-
-    ..  uml::
-
-        package "Some Group" {
-            HTTP - [First Component]
-            [Another Component]
-        }
-
-        node "Other Groups" {
-            FTP - [Second Component]
-            [First Component] --> FTP
-        }
-
-        cloud {
-            [Example 1]
-        }
-
-        database "MySql" {
-            folder "This is my folder" {
-                [Folder 3]
-            }
-            frame "Foo" {
-                [Frame 4]
-            }
-        }
-
-        [Another Component] --> [Example 1]
-        [Example 1] --> [Folder 3]
-        [Folder 3] --> [Frame 4]
-
-Docs: https://plantuml.com/component-diagram
-
-..  index:: Diagrams; Deployment
-
-Deployment diagram
-==================
-
-..  uml::
-
-    artifact Foo1 {
-        folder Foo2
-    }
-
-    folder Foo3 {
-        artifact Foo4
-    }
-
-    frame Foo5 {
-        database Foo6
-    }
-
-    cloud vpc {
-        node ec2 {
-            stack stack
-        }
-    }
+Simple diagrams can be embedded directly into the reStructuredText mark up:
```

Comment:
markup in one word (as a noun):
https://www.merriam-webster.com/dictionary/markup